llms/tests/test_scheduler_config.py

```diff
@@ -0,0 +1,89 @@
+import math
```

Comment:
Thanks for adding tests, that's really great. I do not think we need to make a separate file for this. Could you move tests to `test_lora.py`?